main/NetworkImage/index.tsx

```diff
@@ -156,11 +125,11 @@ function NetworkImage(props: Props): ReactElement {
     <View
       style={[
         {justifyContent: 'center', alignItems: 'center'},
-        shouldFixImageRatio && {aspectRatio: imageRatio},
+        shouldFixImageRatio && {aspectRatio: imageRatio || 110 / 74},
```

Comment:
FYI, by adding default `imageRatio`, it won't confuse the app when trying to render image that is not valid.